src/plugins/postal-savings-rs/converters.ts

```diff
@@ -44,18 +59,49 @@ export function convertAccount (account: AccountDetails, data: string): Account
   }
 }
 
+export function convertCardTransactions (data: string): CardTransaction[] {
+  const transactions: CardTransaction[] = []
+
+  // For some reason all transactions without authorization have positive sign, so make it negative.
+  const fixAmount = (transaction: CardTransaction) => {
+    if (!transaction.authorizationDate && transaction.amount.sum > 0 && !transaction.merchant.startsWith('REFUND ')) {
+      transaction.amount.sum *= -1
```

Comment:
It is a bit dirty. Thinking about better solution for this case.
Maybe it will be better to fix amount on merge with account transactions.